docs/Users/Tutorials/Testing_Bears.rst

```diff
@@ -0,0 +1,194 @@
+How to use LocalBearTestHelper to test your bears
+=================================================
+
+coala has an awesome testing framework so that you can write you tests with
```

Comment:
coala has an awesome testing framework so that you can write tests for bears with ease.